community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -581,7 +581,7 @@ public void transactionRecovered( long txId )
         catch ( Throwable e )
         {
             // Something unexpected happened during startup
-            msgLog.error(
+            msgLog.warn(
```

Comment:
Don't know, it is `warn` in 2.3 as well, so when backporting the change this happened. I'll change this back to `error`.